.eslintrc.cjs

```diff
@@ -18,6 +22,7 @@ module.exports = {
     // see: https://github.com/vuejs/eslint-config-typescript#vueeslint-config-typescriptrecommended
     "@vue/eslint-config-typescript/recommended",
     "@vue/eslint-config-prettier/skip-formatting",
+    "plugin:compat/recommended",
```

Comment:
Is this the line that will check all our code to match the browserlistrc?

Reply:
that's the plugin to check our code, runs as part of `lint:ci:all`